app/graphql/types/compute_profile.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class ComputeProfile < BaseObject
```

Comment:
Could you add queries as well please? It should be just 2 lines in `Types::Query`...

Reply:
True, but I'd like to add tests for the queries as well. If you don't mind, let's do it in a follow up PR.

Reply:
Makes sense.

Reply:
https://projects.theforeman.org/issues/27063